### PR TITLE
fix(ModalFooter): wrap elements in StyledChild only when multiple

### PR DESCRIPTION
**Pull request by mainframev**
Getting back accidentally removed condition for checking if children element is array or not. 

[Slack request](https://skypicker.slack.com/archives/C9B1H0ACW/p1686053579542679)

### CODE REVIEW

**Review by DSil — COMMENTED** [review]
LGTM but pipeline is broken

**Comment by DSil** [comment]
Also, maybe we could add a test to this, so we can avoid a situation like this in the future 💭

**Comment by mainframev** [comment]
> 

yep, will add then 👍🏻

File: packages/orbit-components/src/Modal/ModalFooter/__tests__/index.test.tsx [hunk 1]
```diff
@@ -20,6 +20,16 @@ describe("ModalFooter", () => {
...
   });
 
+  it("should not have StyledChild wrapper on single children", () => {
+    render(
+      <ModalFooter dataTest="footer">
+        <Button type="secondary">Cancel</Button>
+      </ModalFooter>,
+    );
+
+    expect(screen.findByTestId("footer-el-wrapper")).toMatchObject({});
```
Comment (DSil) [inline packages/orbit-components/src/Modal/ModalFooter/__tests__/index.test.tsx:30]: Just for consistency with other tests we have around the repo, why don't we do it like this?

```suggestion
    expect(screen.queryByTestId("footer-el-wrapper")).not.toBeInTheDocument();
```

**Review by DSil — APPROVED** [review]

File: packages/orbit-components/src/Modal/ModalFooter/__tests__/index.test.tsx (hunk 1 above)
Comment (mainframev) [reply packages/orbit-components/src/Modal/ModalFooter/__tests__/index.test.tsx:30]: It won't work for that case

**Pull request merged**